tests/functional/run-action-user-primary-key/source/data/config.json

```diff
@@ -0,0 +1,48 @@
+{
+  "authorization": {
+    "workspace": {
+      "schema": "%env(string:BQ_DATASET)%",
+      "region": "%env(string:BQ_LOCATION)%",
+      "credentials": {
+        "type": "%env(string:BQ_CREDENTIALS_TYPE)%",
+        "project_id": "%env(string:BQ_CREDENTIALS_PROJECT_ID)%",
+        "private_key_id": "%env(string:BQ_CREDENTIALS_PRIVATE_KEY_ID)%",
+        "private_key": "%env(string:BQ_CREDENTIALS_PRIVATE_KEY)%",
+        "client_email": "%env(string:BQ_CREDENTIALS_CLIENT_EMAIL)%",
+        "client_id": "%env(string:BQ_CREDENTIALS_CLIENT_ID)%",
+        "auth_uri": "%env(string:BQ_CREDENTIALS_AUTH_URI)%",
+        "token_uri": "%env(string:BQ_CREDENTIALS_TOKEN_URI)%",
+        "auth_provider_x509_cert_url": "%env(string:BQ_CREDENTIALS_AUTH_PROVIDER_X509_CERT_URL)%",
+        "client_x509_cert_url": "%env(string:BQ_CREDENTIALS_CLIENT_X509_CERT_URL)%"
+      }
+    }
+  },
+  "storage": {
+    "output": {
+      "tables": [
+        {
+          "source": "example",
+          "destination": "out.c-my.example",
+          "primary_key": ["id2"]
```

Comment:
Ja asi nechapu ten test, nikde nevidim, ze by se "id2" nastavilo jako primary key :)

Reply:
Aha, mas pravdu, ono jak jsem to ja mel zalomeny, tak to vypadalo, ze u "id2" je primary_key:false :)

<img width="1717" height="164" alt="Screenshot 2025-07-10 at 13 19 11" src="https://github.com/user-attachments/assets/7db7d795-c53d-4f70-92f2-a96c00fd87e1" />

Reply:
@MiroCillik no ještě že si se zeptal :) ono se to nemá mergovat, ale když jsou nastavený PKs v configu tak platí ty (podle toho jak to děláme v OM - https://github.com/keboola/output-mapping/blob/main/src/Writer/Helper/ConfigurationMerger.php#L38L46)

tak to ještě budu muset upravit!

Reply:
fixnul jsem to, takže pokud jsou v config nastavený PKs tak jsem vezmou z configu, jinak z tabulky v DB